server/game/cards/09.2-BoB/FuryOfTheDamned.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes, Durations } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class FuryOfTheDamned extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Double the base military skill',
+            condition: context => context.game.isDuringConflict(),
+            target: {
+                activePromptTitle: 'Choose bushi characters',
+                numCards: 0,
```

Comment:
I would prefer to see `mode: TargetModes. Unlimited` here rather than `numCards: 0`
It's more forward compatible, and easier to understand

Reply:
Done